src/room007/models/model.py

```diff
@@ -38,6 +44,43 @@ def option_combination_name(options):
         else:
             return [('||| no parameter combinations |||', {option_name: option.default for option_name, option in self.options.items()})]
 
+
+class FeatureManager(object):
+    save = False
+    def __init__(self, fun=None):
+        self.fun = fun
+
+    def __call__(self, *args, **kwargs):
+        data = args[0]
+        feature_name = self._get_feature_name(args, kwargs)
+        if feature_name in data:
+            features = data[feature_name].apply(lambda x: eval(x) if '[' in x else x)
```

Comment:
`series.apply(lambda x: x)` is what I called a no-op `apply` here. I think we should:
- First test if the column contains _a representation of_ structured data or the data itself. (Btw, are lists the only data type that has to be `eval`-ed? Are numbers correctly handled as numbers and not strings here? I guess the `eval`-ing happens when deserializing the feature from its representation loaded from CSV, correct?)
- In the first case (a representation of a list or the like), apply the `eval` transformation. Otherwise, don't do any transformation.

CSV should suffice.